.github/workflows/no-yarn-lock-changes.yml

```diff
@@ -24,8 +24,8 @@ jobs:
       - name: Get file changes
         uses: trilom/file-changes-action@ce38c8ce2459ca3c303415eec8cb0409857b4272
         if: ${{ steps.control.outputs.should_run == 'true' }}
-      - name: Check for yarn.lock changes
+      - name: Check for lockfile changes
         if: ${{ steps.control.outputs.should_run == 'true' }}
         run: |
-          cat $HOME/files.json | jq -e 'any(test("yarn\\.lock$")) | not' \
```

Comment:
Should we add package-lock.json while we're at it?

Reply:
We don't have any package-lock.json's in the vscode repo at the moment